libcodechecker/analyze/plist_parser.py

```diff
@@ -3,170 +3,185 @@
 #   This file is distributed under the University of Illinois Open Source
 #   License. See LICENSE.TXT for details.
 # -------------------------------------------------------------------------
-""""""
+"""
+Parse the plist output of an analyzer and convert it to a report for
+further processing.
 
-# FIXME: This file contains some workarounds.
-# Remove them as soon as a proper clang version comes out.
-
-import plistlib
-from xml.parsers.expat import ExpatError
-
-from libcodechecker.analyze import plist_helper
-from libcodechecker.logger import LoggerFactory
-
-LOG = LoggerFactory.get_new_logger('PLIST_PARSER')
+With the newer clang releases more information is available in the plist files.
 
+* Before Clang v3.7:
```

Comment:
Clang 3.7 was released in 2015 Sept. Do we want to support the earlier versions? (Clang 3.6 was released in 2015 feb.), 3.8 is available in ubuntu 16.04 LTS.  And 3.8 is also available in the 14.04 backports repository.